xml/adm_sudo.xml

```diff
@@ -0,0 +1,611 @@
+                  The default is ON.
+                </para>
+              </entry>
+              <entry>
+                <screen>Defaults env_reset # Turn env_reset flag ON</screen>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <literal>env_keep</literal>
+              </entry>
+              <entry>
+                <para>
+                  List of environment variables to keep when the <literal>env_reset</literal> flag is ON.
+                </para>
+              </entry>
+              <entry>
+                <screen>
+# Set env_keep to contain EDITOR and PROMPT
+Defaults env_keep = "EDITOR PROMPT"
+Defaults env_keep += "JRE_HOME" # Add JRE_HOME
+Defaults env_keep -= "JRE_HOME" # Remove JRE_HOME
+</screen>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <literal>env_delete</literal>
+              </entry>
+              <entry>
+                <para>
+                  List of environment variables to remove when the <literal>env_reset</literal> flag is OFF.
+                </para>
+              </entry>
+              <entry>
+                <screen>
+# Set env_delete to contain EDITOR and PROMPT
+Defaults env_delete = "EDITOR PROMPT"
+Defaults env_delete += "JRE_HOME" # Add JRE_HOME
+Defaults env_delete -= "JRE_HOME" # Remove JRE_HOME
+</screen>
+              </entry>
+            </row>
+          </tbody>
+        </tgroup>
+      </table>
+
+      <para>
+        The <literal>Defaults</literal> token can also be used to create aliases for a collection
+        of users, hosts and commands.
+        Furthermore, it is possible to apply an option setting only to a specific set of users.
+      </para>
+
+      <para>
+        For detailed information about the <filename>/etc/sudoers</filename> configuration file, consult
+        <command>man 5 sudoers</command>.
+      </para>
+    </sect2>
+
+    <sect2 xml:id="sec.sudo.conf.rule">
+      <title>Rules in sudoers</title>
+
+      <para>
+        Rules in sudoers can be very complex, so this section will only touch the basics.
+        Each rule follows the basic scheme (<literal>[]</literal> marks optional parts):
+      </para>
+
+      <screen>
+#Who      Where         As whom      Tag                What
+User_List Host_List = [(User_List)] [NOPASSWD:|PASSWD:] Cmnd_List
+</screen>
+
+      <variablelist>
+        <title>Syntax for sudoers rules</title>
+        <varlistentry>
+          <term><literal>User_List</literal></term>
+          <listitem>
+            <para>
+              One or more (separated by <literal>,</literal>) identifiers:
+              Either a user name, a group in the format <literal>%GROUPNAME</literal>
+              or a user ID in the format <literal>#UID</literal>.
+              Each identifier can be prefixed by <literal>!</literal> for negation.
```

Comment:
Change sentence order: Goal first, then action. Makes it more logical/easier to read.

"To negate an identifier prefix it with !."